src/backend/base/langflow/components/models/ollama.py

```diff
@@ -249,30 +258,58 @@ async def update_build_config(self, build_config: dict, field_value: Any, field_
 
         return build_config
 
-    async def get_model(self, base_url_value: str, tool_model_enabled: bool | None = None) -> list[str]:
+    async def get_non_embedding_models(self, base_url_value: str, tool_model_enabled: bool | None = None) -> list[str]:
```

Comment:
```suggestion
    async def get_models(self, base_url_value: str, tool_model_enabled: bool | None = None) -> list[str]:
```

Because this component is a subclass of the LC model component, it's known/ assumed that the models are language models.
Also, we can standardise this function name as an abstract method in the future, so all new providers can have their custom implementations.